game_eggs/steamcmd_servers/assetto_corsa/egg-assetto-corsa.json

```diff
@@ -91,7 +97,8 @@
             "default_value": "8081",
```

Comment:
dont provide a port, because this is part of the admin

Reply:
?

Reply:
I removed the default value and set it to nullable